src/containers/SearchPage/SearchPage.duck.js

```diff
@@ -114,13 +114,19 @@ export const searchListings = (searchParams, config) => (dispatch, getState, sdk
   //       ...and then turned enforceValidListingType config to true in configListing.js
   // Read More:
   // https://www.sharetribe.com/docs/how-to/manage-search-schemas-with-flex-cli/#adding-listing-search-schemas
-  const searchValidListingTypes = listingTypes => {
+  const searchValidListingTypes = (listingTypes, listingTypePathParam, isListingTypeVariant) => {
+    const isValidListingTypeParam =
+      listingTypePathParam && listingTypes.find(lt => lt.listingType === listingTypePathParam);
     return config.listing.enforceValidListingType
       ? {
           pub_listingType: listingTypes.map(l => l.listingType),
           // pub_transactionProcessAlias: listingTypes.map(l => l.transactionType.alias),
           // pub_unitType: listingTypes.map(l => l.transactionType.unitType),
         }
+      : isListingTypeVariant && isValidListingTypeParam
+      ? {
+          pub_listingType: listingTypePathParam,
+        }
```

Comment:
I think this needs to be swapped.
1. "isListingTypeVariant && isValidListingTypeParam" => a single valid listing type in use
2. config.listing.enforceValidListingType => the previous behaviour

```suggestion
    return isListingTypeVariant && isValidListingTypeParam
      ? {
          pub_listingType: listingTypePathParam,
        }
      : config.listing.enforceValidListingType
      ? {
          pub_listingType: listingTypes.map(l => l.listingType),
          // pub_transactionProcessAlias: listingTypes.map(l => l.transactionType.alias),
          // pub_unitType: listingTypes.map(l => l.transactionType.unitType),
        }
```